objects/c932e8/script.lua

```diff
@@ -1,5 +1,36 @@
-function onLoad()
-    if not Global.getVar("gameStarted") then
+local initialized = false
+local serpent = nil
+local elementScanZone = nil
+
+local yHeight
+
+local slumberTrack = {[0]=5,7,8,10,11,12,13}
+local slumberPoints = {
+    Vector(1.14, 0.20, 0.65),
+    Vector(0.92, 0.20, 0.65),
+    Vector(0.70, 0.20, 0.65),
+    Vector(1.25, 0.20, 0.84),
+    Vector(1.03, 0.20, 0.84),
+    Vector(0.81, 0.20, 0.84),
+}
+
+function onLoad(saved_data)
+    if saved_data ~= "" then
+        local loaded_data = JSON.decode(saved_data)
+        if loaded_data.serpent then
```

Comment:
you already reference serpents guid later on in the code, so you should probably just extract that guid out and then reference it here rather than storing it in json